specification/managedservices/resource-manager/Microsoft.ManagedServices/preview/2020-02-01-preview/managedservices.json

```diff
@@ -1044,24 +1044,24 @@
         "roleDefinitionId"
       ],
       "type": "object",
-      "description": "Authorization tuple containing principal Id (of user/service principal/security group) and role definition id."
+      "description": "The user ID and permanent role for a principal in the managedBy tenant."
```

Comment:
>permanent role [](http://example.com/codeflow?start=38&length=14)

there is no concept like "permanent role"

Reply:
Good point - changed